examples/python_side_metrics.py

```diff
@@ -76,11 +76,12 @@ def forward(self, data: List[int]) -> List[bool]:
 
 
 if __name__ == "__main__":
-    # Run the metrics server in another thread.
-    metric_thread = threading.Thread(target=metric_service, daemon=True)
-    metric_thread.start()
+    # Run the metrics server in another process.
+    metric_process = mp.Process(target=metric_service, daemon=True)
+    metric_process.start()
```

Comment:
Maybe this is not a good example to use the daemon process.

1. Python side metrics are not crucial to the model serving.
2. Thread can save a lot of resource usage.